### PR TITLE
Update EANet to Keras 3

### PR DESCRIPTION
**Pull request by nkovela1**

### CODE REVIEW

File: examples/vision/md/eanet.md [hunk 1]
```diff
@@ -73,7 +67,7 @@ learning_rate = 0.001
 label_smoothing = 0.1
 validation_split = 0.2
 batch_size = 128
-num_epochs = 50
+num_epochs = 1  # Recommended num_epochs = 1.
```
Comment (fchollet) [inline examples/vision/md/eanet.md:70]: This should be 50

**Review by fchollet — COMMENTED** [review]
Thanks for the PR

File: examples/vision/md/eanet.md (hunk 1 above)
Comment (nkovela1) [reply examples/vision/md/eanet.md]: Fixed the comment thanks!

**Review by fchollet — APPROVED** [review]
LGTM, thank you

**Pull request merged**